### PR TITLE
Training 03 databases

### PR DESCRIPTION
**Pull request by danetsao**

### CODE REVIEW

File: src/main/resources/db/changes/0010-add-available-column.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      changes:
+        - addColumn:
+            tableName: books
+            columns:
+            - column:
+                name: available
+                type: BOOLEAN
+                constraints:
+                  nullable: true
+                  defaultValueBoolean: true
```
Comment (ncovercash) [inline src/main/resources/db/changes/0010-add-available-column.yaml:14]: Ah good catch, would you mind editing the training document with that instead of `defaultValue`

File: src/test/java/org/folio/sample/integration/api/books/BookAvailableTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.hasSize;
+
+
+
+class BookAvailableTest extends AbstractBaseApiTest {
+
+  @MethodSource
```
Comment (ncovercash) [inline src/test/java/org/folio/sample/integration/api/books/BookAvailableTest.java]: Why is this here?

File: src/test/java/org/folio/sample/integration/api/books/BookAvailableTest.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+    // Assert the list is empty
+    List<BookDTO> collection = response
+      .getBody()
+      .as(new TypeRef<List<BookDTO>>() {});
+    assertThat(collection, hasSize(0));
+  }
+
+  @Test
+  void testNotFound() {
```
Comment (ncovercash) [inline src/test/java/org/folio/sample/integration/api/books/BookAvailableTest.java]: Looks like these and a few others were copied and don't need to be here

**Review by ncovercash — CHANGES_REQUESTED** [review]

File: src/test/java/org/folio/sample/integration/api/books/BookAvailableTest.java (hunk 2 above)
Comment (danetsao) [reply src/test/java/org/folio/sample/integration/api/books/BookAvailableTest.java]: Yea that was also copied over from the other test, but I should have removed all of that now.

File: src/test/java/org/folio/sample/integration/api/books/BookAvailableTest.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      .post(getRequestUrl("books"))
+      .then()
+      .statusCode(is(HttpStatus.CREATED.value()));
+
+    ra()
+      .body(
+        BookForCreationDTO
+          .builder()
+          .name("Book 2")
+          .publishedDate(LocalDate.of(2000, 1, 1))
```
Comment (ncovercash) [inline src/test/java/org/folio/sample/integration/api/books/BookAvailableTest.java:57]: I might add .available(false) on one of these to check that it isn't there (and explicitly provide .available(true) on the other)

**Review by ncovercash — CHANGES_REQUESTED** [review]

**Review by ncovercash — APPROVED** [review]
LGTM
